OMEdit/OMEdit/OMEditGUI/Simulation/SimulationDialog.cpp

```diff
@@ -2175,44 +2174,45 @@ void SimulationDialog::showArchivedSimulation(QTreeWidgetItem *pTreeWidgetItem)
  */
 void SimulationDialog::simulate()
 {
-  if (validate()) {
-    // interactive simulation
-    if (mpInteractiveSimulationGroupBox->isChecked() || mIsReSimulate) {
+  if (!validate()) {
+    return;
+  }
+  // if no option is selected then show error message to user
+  if (!(mpSaveExperimentAnnotationCheckBox->isChecked() ||
```

Comment:
This check was never executed before running interactive simulations, I rewrote the logic to make the nesting not as deep and that we do this check for both simulation possibilities